src/shared/AsyncTypeaheadSelect/AsyncTypeaheadSelect.test.tsx

```diff
@@ -13,33 +13,29 @@ const {
 
 describe("Async typeahead", () => {
   jest.useFakeTimers();
-  it("should render a select with async typeahead suggestions", async () => {
+  it("It should render an async typeahead", async () => {
```

Comment:
Nitpick, no need to write "It" in the description since it's already there in the code